hardware/requirements.txt

```diff
@@ -0,0 +1 @@
+Pillow==5.2.0
```

Comment:
Why is this needed?

Reply:
It's a requirement to use an ImageField in django (https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ImageField). This allows to upload an image through the admin. I'm not sure as to why is needed for the ImageField but not the FileField

Reply:
What's the right way to install this dependency btw? Running `pip install -r requirements.txt` from the project root won't include the dependencies of the apps like this one. I guess we should just add a step to the readme and/or provide a script that runs pip install for all the requirements.

Reply:
I didn't include it in the root requirements.txt because it should be easy to include/exclude the hardware lab. I think that it should be enough stating the steps needed to use the hardware lab in the readme (install hardware/requirements, uncomment the app INSTALLED_APPS, uncomment the line in the navbar...)

Reply:
OR we could provide a custom manage command to include/exclude apps. I don't know how that would work though